modules/end-user-guide/partials/proc_provisioning-jetbrains-activation-code-for-offline-use.adoc

```diff
@@ -1,55 +1,67 @@
 [id="provisioning-jetbrains-activation-code-for-offline-use_{context}"]
 = Provisioning JetBrains activation code for offline use
 
-An activation code for offline use is a file with a license code that can be retrieved from the license management section of your JetBrains account for the license that is assigned to you. When you purchase a personal subscription or are assigned a commercial subscription by your organization, you receive an email prompting you to create a JetBrains account that becomes connected with the license.
-
-NOTE: When using an activation code to activate a product, you need to generate a new activation code and apply it to your product each time the subscription is renewed.
+This section describes how to provision the JetBrains activation code to {prod-short}, when using link:https://www.jetbrains.com/store/[a JetBrains IDE requesting a registration]. After each subscription renewal, you need to generate an activation code and apply it to your product.
 
 .Prerequisites
 
-* JetBrains account
-* Personal or organization subscription
+* An active link:https://account.jetbrains.com/login[JetBrains account].
+* An active link:https://www.jetbrains.com/store/comparison.html[personal or organization subscription].
```

Comment:
```suggestion
* An active link:https://sales.jetbrains.com/hc/en-gb[subscription for organizations or for individual use].
```

Reply:
just aligning the terminology with https://www.jetbrains.com/store/#commercial?billing=yearly

Reply:
OK but ket'sd keep the link to the www website.

```suggestion
* An active link:https://www.jetbrains.com/store/comparison.html[subscription for organizations or for personal use].
```